CMakeLists.txt

```diff
@@ -306,6 +306,8 @@ install(
   COMPONENT library
 )
 
+install(DIRECTORY include/ DESTINATION include/ FILES_MATCHING PATTERN "*.h" PATTERN "*.hpp")
```

Comment:
This command copies the content of the `include` directory of the repository into an `include` directory of the `CMAKE_INSTALL_PREFIX`, so it is fine as long as we keep the current structure in which all the headers are inside an `opengv` directory.